### PR TITLE
Added support for MICROSECOND

### PR DESCRIPTION
**Pull request by Kowsikraja**

### CODE REVIEW

**Review by Katakam-Phani-Mahidhar — APPROVED** [review]

File: core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java [hunk 1]
```diff
@@ -2069,6 +2069,15 @@ public class SqlStdOperatorTable extends ReflectiveSqlOperatorTable {
   public static final SqlDatePartFunction SECOND =
       new SqlDatePartFunction("SECOND", TimeUnit.SECOND);
 
+  /**
+   * The SQL <code>MICROSECOND</code> operator. Returns the Second
```
Comment (dm-shantanuc) [inline core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java:2073]: It should be returns Microseconds, not second

**Review by dm-shantanuc — APPROVED** [review]
LGTM

File: core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java (hunk 1 above)
Comment (Kowsikraja) [reply core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java]: @dm-shantanuc i had resolved this.Please have a look

**Review by dm-shantanuc — APPROVED** [review]
LGTM

**Pull request merged**